client/src/app/site/login/components/login-mask/login-mask.component.ts

```diff
@@ -134,11 +134,9 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
             ();
 
         // Maybe the operator changes and the user is logged in. If so, redirect him and boot OpenSlides.
-        this.operatorSubscription = this.operator.operatorUpdatedEvent.subscribe(user => {
-            if (user) {
-                this.clearOperatorSubscription();
-                this.authService.redirectUser(this.currentMeetingId);
-            }
+        this.operatorSubscription = this.operator.operatorUpdatedEvent.subscribe(() => {
+            this.clearOperatorSubscription();
+            this.authService.redirectUser(this.currentMeetingId);
         });
```

Comment:
Why removed the condition?

Reply:
`operator.operatorUpdatedEvent` only emitted void. eslint was detecting that and marked it as an error.

https://github.com/OpenSlides/openslides-client/blob/51aa6d5c8862a3cb2e6e9027499eb848316742e7/client/src/app/core/core-services/operator.service.ts#L116

Reply:
Okay.